Essentials/src/main/java/com/earth2me/essentials/utils/LocationUtil.java

```diff
@@ -274,7 +275,8 @@ public static boolean shouldFly(IEssentials ess, final Location loc) {
             }
         }
 
-        return y < 0;
+        //If not then check if player is in the void
```

Comment:
Sorry, can you also fix this comment :)

Reply:
Ahah yes it is already fixed, I've read your comment like a computer, I realized too late that I did forget this one :D

Reply:
No problem, thanks